spec/run_results_combiner_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require "helper"
+
+if SimpleCov.usable?
+  describe SimpleCov::RunResultsCombiner do
+    describe "with two faked coverage resultsets" do
+      before do
+        @resultset1 = {
+          source_fixture("sample.rb") => {
+            :lines => [nil, 1, 1, 1, nil, nil, 1, 1, nil, nil],
+            :branches => {[:if, 3, 8, 6, 8, 36] => {[:then, 4, 8, 6, 8, 12] => 47, [:else, 5, 8, 6, 8, 36] => 24}},
+          },
+          source_fixture("app/models/user.rb") => {
+            :lines => [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil],
+            :branches => {[:if, 3, 8, 6, 8, 36] => {[:then, 4, 8, 6, 8, 12] => 47, [:else, 5, 8, 6, 8, 36] => 24}},
+          },
+          source_fixture("app/controllers/sample_controller.rb") => {:lines => [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil]},
+          source_fixture("resultset1.rb") => {:lines => [1, 1, 1, 1]},
```

Comment:
is it common that there will be a `:lines` key but no `:branches` key? I'd ahve expected both of them to always be present